snap/snapcraft.yaml

```diff
@@ -133,12 +126,11 @@ apps:
   # this service exits. In the Docker version, the customized redis' entrypoint.sh performs
   # the similar actions as described above.
   security-bootstrapper-redis:
-    adapter: none
     after:
       - security-secretstore-setup
     command: bin/setup-redis-acl.sh
     command-chain:
-      - bin/service-config-overrides.sh
+      - bin/source-env-file.sh
     environment:
       ENV_FILE: $SNAP_DATA/config/security-bootstrapper/res/security-bootstrapper.env
```

Comment:
`ENV_FILE` should be changed to `$SNAP_DATA/config/security-bootstrapper/overrides.env`